webapp/sso.py

```diff
@@ -1,11 +1,10 @@
-import os
 import flask
 import socket
 from django_openid_auth.teams import TeamsRequest, TeamsResponse
 from flask_openid import OpenID
 
 SSO_LOGIN_URL = "https://login.ubuntu.com"
-SSO_TEAM = os.getenv("OPENID_LAUNCHPAD_TEAM", "canonical")
+SSO_TEAM = "canonical-webmonkeys"
```

Comment:
Was this the fix to the infinite redirect? I'd say we definitely want this to be at least company public. If it's the difference between `canonical` and `canonical-webmonkeys` (i.e. the difference between public and private launchpad teams) that make this work, then we should talk to @Roadmaster about getting our sites permission to use the private `canonical` group.

Reply:
No the issue with the redirect, was missing dependencies in the container. Using the `canonical` group does not work at the moment and `canonical-webmonkeys` sounded like a reasonable compromise as everyone in the team is usually added to it.

Reply:
If @nottrobin is happy, feel free to press the merge button. I think the other issue about the `canonical` group can be tackled separately.